### PR TITLE
Dont fail xsiam format

### PR DESCRIPTION
**Pull request by JudahSchwartz**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes:[ link to the issue](https://jira-hq.paloaltonetworks.local/browse/CIAC-2646)

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Must have
- [ ] Tests
- [ ] Documentation
- [ ] [VS Code Extension](https://docs.gitlab.com/ee/ci/yaml/#tags)
  - [ ] Functionality implemented in the extension - <u>\<link-to-pr-here\></u>
  - [ ] N/A - Functionality cannot be implemented in the extension because <u>\<add-reason-here\></u>

### CODE REVIEW

**Comment by JudahSchwartz** [comment]
@dorschw after looking at the code for format where I thought there mightve been some bugs because of the id, looks like none of the files which store ID differently use that code at all at this point, so I dont think theres any bugs

I caused the format command not to throw exceptions when theres a new xsiam entity present, but when it comes to implementing the format/validate commands themselves itll need to be done on an entity by entity basis when we get to it

File: demisto_sdk/commands/common/tools.py [hunk 1]
```diff
@@ -1798,12 +1798,11 @@ def _get_file_id(file_type: str, file_content: Dict):
...
     """
-    file_id = ''
     if file_type in ID_IN_ROOT:
-        file_id = file_content.get('id', '')
+        return file_content.get('id', '')
     elif file_type in ID_IN_COMMONFIELDS:
-        file_id = file_content.get('commonfields', {}).get('id')
-    return file_id
+        return file_content.get('commonfields', {}).get('id')
+    return file_content.get('trigger_id', '')
```
[the remark this answers is not in the document]
Comment (JudahSchwartz) [reply demisto_sdk/commands/common/tools.py:1805]: current behavior returns '' actually. (line 1801). Should I still change?

[the remark this answers is not in the document]
Comment (dorschw) [reply demisto_sdk/commands/common/tools.py:1805]: no, it's fine

File: demisto_sdk/commands/format/format_module.py [hunk 2]
```diff
@@ -291,8 +291,13 @@ def run_format_on_file(input: str, file_type: str, from_version: str, interactiv
...
         del kwargs['id_set_path']
-    update_object = FILE_TYPE_AND_LINKED_CLASS[file_type](input=input, path=schema_path, from_version=from_version,
-                                                          interactive=interactive, **kwargs)
+    updater_class = FILE_TYPE_AND_LINKED_CLASS.get(file_type)
+    if not updater_class:  # fail format so long as xsiam entities dont have formatters
+        print_warning(f'No  updater_class was found for file type {file_type}')
+        return logger(input, 1, VALIDATE_RES_SKIPPED_CODE)
+
+    update_object = updater_class(input=input, path=schema_path, from_version=from_version,
+                                  interactive=interactive, **kwargs)
```
[the remark this answers is not in the document]
Comment (dorschw) [reply demisto_sdk/commands/format/format_module.py:300]: alright, let's keep it this way

**Review by dorschw — APPROVED** [review]

**Pull request merged**